pyproject.toml

```
@@ -133,6 +133,24 @@ version_files = [
     'docs/package.json:^  "version":',
 ]
 
+[tool.commitizen.customize]
+commit_parser = '^(?P<change_type>feat|fix|refactor|perf|BREAKING CHANGE|docs)(?:\((?P<scope>[^()\r\n]*)\)|\()?(?P<breaking>!)?:\s(?P<message>.*)?'
+change_type_order = [
+  "BREAKING CHANGE",
+  "✨ New",
+  "🐛 Fixes",
+  "⚙️ Under the Hood",
+  "⚡ Performance",
```

For consistency with "Documentation Improvements":

```suggestion
  "⚡ Performance Improvements",
```